content/en/docs/reference/_index.md

```diff
@@ -35,7 +35,7 @@ client libraries:
 ## CLI Reference
 
 * [kubectl](/docs/reference/kubectl/overview/) - Main CLI tool for running commands and managing Kubernetes clusters.
-    * [JSONPath](/docs/reference/kubectl/jsonpath/) - Syntax guide for using [JSONPath expressions](http://goessner.net/articles/JsonPath/) with kubectl.
```

Comment:
I think this might have been indented to show that JSONPath with kubectl is a topic that's of interest only to a subset of people who are interested in `kubectl`.